diff --git a/dist/esm/LexicalComposerContext.js b/dist/esm/LexicalComposerContext.js
index 9d6f15aabcc030e059eabf87a5e6bf47b1463ae9..f29186976c1a7edfa6fbf3f0685ad9d97458d368 100644
--- a/dist/esm/LexicalComposerContext.js
+++ b/dist/esm/LexicalComposerContext.js
@@ -18,11 +18,15 @@ function createLexicalComposerContext(parent, theme) {
 const LexicalComposerContext = createContext(null);
 const useLexicalComposerContext = () => {
   const composerContext = useContext(LexicalComposerContext);
-  if (!composerContext) {
-    {
-      throw Error(`useLexicalComposerContext: cannot find a LexicalComposerContext`);
+
+  if (import.meta.env.DEV) {
+    if (!composerContext) {
+      {
+        throw Error(`useLexicalComposerContext: cannot find a LexicalComposerContext`);
+      }
     }
   }
+
   return composerContext;
 };
 
